README.md

```diff
@@ -24,6 +24,13 @@ Test data is freely available from [OpenSlide](http://openslide.cs.cmu.edu/downl
 
 To see all available options, run: `wsi2dcm --help`.
 
+## Additional tags
```

Comment:
I like your suggestion of just putting the output of wsi2dcm --help here, so users who are evaluating have a quick way to see the feature set.

Reply:
i'll update this PR as https://github.com/GoogleCloudPlatform/wsi-to-dicom-converter/pull/33 will be merged to have complete set of params